i18n.php

```diff
@@ -20,6 +21,7 @@
     'bg'    => array(
         'snippet_context_choice'  => 'В сет <snippet_undefined><snippet_keyword>%count%</snippet_keyword> има недекларирани стъпки. Изберете в кой Context да бъдат създадени:</snippet_undefined>',
         'snippet_proposal_title'  => '<snippet_undefined><snippet_keyword>%count%</snippet_keyword> има липсващи стъпки. Можете да ги създадете чрез:</snippet_undefined>',
+        'snippet_proposal_use'    => 'Не забравяйте тези %count% use изрази:',
```

Comment:
I didn't know if I should translate to other languages or not. So I used a translation tool to get a generic translated sentence for each language (except for English and French, which use plural correctly).

Reply:
Brilliant, thanks - I can't really check any of the languages but the English & French look right and the German looks convincing :laughing: